mne/time_frequency/ar.py

```diff
@@ -149,7 +149,6 @@ def iir_filter_raw(raw, order, picks, tmin=None, tmax=None):
     a : array
         filter coefficients
     """
-    picks = picks[:5]
```

Comment:
@agramfort I assume this was a bug? I noticed the example had a `[:5]` in it so I assume this is left over from debugging?